### ISSUE TITLE
Support for kotlin

### ISSUE THREAD

**Issue by antechrestos** [novoda/bintray-release#134]
Hello, 

First of all thank you for your work as it eased a lot our publication.

We've just implemented a kotlin library and we spotted a "lack" : as observed, it does not trigger pom, source and java doc generation. I am not sure it is a lack from your side or the maven plugin you rely on.

Dry run :
```
$> ./gradlew -m --warn library:clean library:build  library:bintrayUpload
:library:clean SKIPPED
:library:compileKotlin SKIPPED
:library:compileJava SKIPPED
:library:processResources SKIPPED
:library:classes SKIPPED
:library:jar SKIPPED
:library:assemble SKIPPED
:library:compileTestKotlin SKIPPED
:library:compileTestJava SKIPPED
:library:processTestResources SKIPPED
:library:testClasses SKIPPED
:library:test SKIPPED
:library:check SKIPPED
:library:build SKIPPED
:library:bintrayUpload SKIPPED
```

To force the generation of maven artifacts, I have to force it by adding `publishToMavenLocal` task

```
./gradlew -m --warn library:clean library:build  library:publishToMavenLocal library:bintrayUpload
Parallel execution with configuration on demand is an incubating feature.
:library:clean SKIPPED
:library:compileKotlin SKIPPED
:library:compileJava SKIPPED
:library:processResources SKIPPED
:library:classes SKIPPED
:library:jar SKIPPED
:library:assemble SKIPPED
:library:compileTestKotlin SKIPPED
:library:compileTestJava SKIPPED
:library:processTestResources SKIPPED
:library:testClasses SKIPPED
:library:test SKIPPED
:library:check SKIPPED
:library:build SKIPPED
:library:generatePomFileForMavenPublication SKIPPED
:library:mavenJavadocJar SKIPPED
:library:mavenSourcesJar SKIPPED
:library:publishMavenPublicationToMavenLocal SKIPPED
:library:publishToMavenLocal SKIPPED
:library:bintrayUpload SKIPPED
```

The result, if I don't force maven artifact generation, is that it will upload a jar to bintray without the pom. Projects that will import it will miss the dependencies.

**Comment by StefMa** [comment]
Thank you for spotting this issue.

The issue is not fixed by running "publishToMavenLocal". Rather than executing explicitly "generatePomFile...", "mavenJavadocJar" and "mavenSourcesJar"..

I ran into the same issue with a android library in some cases. See https://github.com/StefMa/AndroidArtifacts/issues/8.

Can you share some "project details". How many modules to you have etc?

**Comment by antechrestos** [comment]
@StefMa Sorry for the late reply, I have been quite busy and forgot to answer.
I have this issue in the [following project](https://github.com/Orange-OpenSource/datashare-android-lib) . I can produce a smaller one if you want.

**Comment by StefMa** [comment]
Should be already fixed by #160 
Have nothing todo with Kotlin.

If you run into the same issue again feel free to open a new issue 👍

**Comment by antechrestos** [comment]
@StefMa Thanks. I will wait for the next release then. Cheers!
